packages/text-editor/src/components/TextEditor.svelte

```diff
@@ -55,6 +55,15 @@
   let placeHolderStr: string = ''
 
   $: ph = translate(placeholder, {}, $themeStore.language).then((r) => {
+    if (editor !== undefined && placeHolderStr !== r) {
```

Comment:
better to start translation after editor is defined

Reply:
editor must be defined with placeholder plugin configured with translated string. these changes is a dirty hack that updates plugin existing state when translation changes

Reply:
Nope, we need to translate even before the editor is defined – placeholder is set when the editor is defined, so it's required there